README.md

```diff
@@ -1,35 +1,51 @@
 
 ## Pipeline summary
 
-<!-- TODO: Fill in short bullet-pointed list of the default steps in the pipeline -->
+This workflow only supports PacBio Hifi reads in `.fastq.` format for producing an assembly and QC stats. Each set of reads are assembled with `flye`, assembly statistics summarized with `QUAST`, reads mapped back to the assembly with `minimap2` and lineage specific QC stats produced with `BUSCO`. The workflow then reports the QC stats from `QUAST` and `BUSCO` into an `.html` report with `MultiQC`.
 
 ## Quick start
 
-<!-- TODO: Fill in short bullet-pointed list of the default steps to get the pipeline up and running -->
+1. Install [`Nextflow`](https://www.nextflow.io/docs/latest/getstarted.html#installation) (`>=22.10.1`)
+
+2. Install [`Docker`](https://docs.docker.com/engine/installation/), [`Singularity`](https://www.sylabs.io/guides/3.0/user-guide/) (you can follow [this tutorial](https://singularity-tutorial.github.io/01-installation/)), or [`Conda`](https://conda.io/miniconda.html). You can use conda to install Nextflow itself but use it for managing software within pipelines as a last resort. We recommend using `Docker` if possible as this has been tested most frequently. See the nf-core [docs](https://nf-co.re/usage/configuration#basic-configuration-profiles)) for more information.
+
+3. Test the pipeline on a minimal test dataset with a single command:
+
+   ```console
+   nextflow run Arcadia-Science/hifi2genome -profile test,YOURPROFILE --outdir <OUTDIR>
+   ```
 
-## Full documentation
+   Note that some form of configuration will be needed so that Nextflow knows how to fetch the required software. This is usually done in the form of a config profile (`YOURPROFILE` in the example command above). You can chain multiple config profiles in a comma-separated string.
 
-<!-- TODO: Fill in this section with how to fully use the pipeline, what the inputs are and what the outputs look like. If this section ends up being super long, feel free to create a new docs/ directory and add details there. -->
+   > - The pipeline comes with several config profiles, but we recommend using `docker` when possible, such as `-profile test,docker`.
+   > - Please check [nf-core/configs](https://github.com/nf-core/configs#documentation) to see if a custom config file to run nf-core pipelines already exists for your Institute. If so, you can simply use `-profile <institute>` in your command. This will enable either `docker` or `singularity` and set the appropriate execution settings for your local compute environment.
```

Comment:
```suggestion
   > - Please check [nf-core/configs](https://github.com/nf-core/configs#documentation) to see if a custom config file to run nf-core pipelines already exists for your institute. If so, you can simply use `-profile <institute>` in your command. This will enable either `docker` or `singularity` and set the appropriate execution settings for your local compute environment.
```